.ci/openvino-onnx/Dockerfile

```diff
@@ -0,0 +1,94 @@
+FROM ubuntu:20.04
+
+LABEL version=2020.07.03.2
+
+ARG http_proxy
+ARG https_proxy
+ENV http_proxy ${http_proxy}
+ENV https_proxy ${https_proxy}
+
+ENV CI=true
+ENV DEBIAN_FRONTEND=noninteractive
+ENV PYTHONUNBUFFERED 1
+
+# Install base dependencies
+RUN apt-get update && apt-get install -y locales && apt-get clean autoclean && apt-get autoremove -y
+
+# Set the locale to en_US.UTF-8
+RUN locale-gen en_US.UTF-8
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
+
+# OpenVINO dependencies
+RUN apt-get update && apt-get -y --no-install-recommends install \
+        autoconf \
+        automake \
+        build-essential \
+        cmake \
+        curl \
+        git \
+        libtool \
+        ocl-icd-opencl-dev \
+        pkg-config \
+        unzip \
+        wget && \
+  apt-get clean autoclean && \
+  apt-get autoremove -y
+
+# Python dependencies
+RUN apt-get -y --no-install-recommends install \
+        python3 \
+        python3-pip \
+        python3-dev \
+        python3-virtualenv \
+        cython3 \
+        tox && \
+    apt-get clean autoclean && \
+    apt-get autoremove -y
+
+# ONNX dependencies
+RUN apt-get -y --no-install-recommends install \
+        git-lfs \
+        protobuf-compiler \
+        libprotobuf-dev && \
+    apt-get clean autoclean && \
+    apt-get autoremove -y
+
+# Build OpenVINO
+COPY . /openvino/
+WORKDIR /openvino/build
+RUN cmake .. \
+    -DCMAKE_BUILD_TYPE=Debug \
```

Comment:
Are we sure we want Debug version in CI? It would be slower...

Reply:
Changed to Release.